### ISSUE TITLE
URL in project description shows 404

### ISSUE THREAD

**Issue by opreaadrian** [Zhouzi/TheaterJS#4]
Hello,

I was looking at the project and wanted to see a demo of it but unfortunately I found an issue.
http://gabinaureche.com/TheaterJS is throwing a 404 error.

This also messes up the example on CodePen.

**Comment by Zhouzi** [comment]
Hello @opreaadrian,

Could you please try to access http://gabinaureche.com/TheaterJS again? 

I'm not experiencing any error from my side. Maybe it has been down for a little, I'd eventually get in touch with my host.

Anyway, thank you for the heads-up!

**Comment by Zhouzi** [comment]
Thank you for reporting it @flenter! Did you come across a 404 page also?

Sadly, I can't reproduce it:
http://www.downforeveryoneorjustme.com/http://gabinaureche.com/TheaterJS/
I will contact my host to know what's going wrong. 

For now, I replaced the project's url by the codepen one: http://codepen.io/Zhouzi/full/JoRazP/

**Comment by flenter** [comment]
Indeed, I got a 404 on the http://gabinaureche.com/TheaterJS link too.

Unfortunately, the codepen version also doesn't work since it relies on javascript (http://gabinaureche.com/TheaterJS/build/theater.js) and css (http://gabinaureche.com/TheaterJS/build/styles.css ) for which the server also responded with 404's

**Comment by opreaadrian** [comment]
Hello guys,

From what I saw, it's only acting funny in browsers that use webkit/blink -- Chrome, Safari, Opera. It's working fine in Firefox and Safari mobile.

http://youtu.be/fXC8eS9ahNc

Cheers!

**Comment by Zhouzi** [comment]
Thank you @flenter and thank you very much @opreaadrian for taking the time to record it, much appreciated! :)

Did you guys accessed to the page once? Or it has been down all the time?
I am wondering if the server experienced some down time periods or if the issue is about something else.

I contacted my web hosting company to know what's happening. 
Until then, I embedded the source code on Codepen so it's accessible for everyone: 

http://codepen.io/Zhouzi/full/JoRazP/
